packages/client/tests/functional/_utils/setupTestSuiteMatrix.ts

```diff
@@ -4,6 +4,11 @@ import { dropTestSuiteDatabase, setupTestSuiteDbURI } from './setupTestSuiteEnv'
 
 export type TestSuiteMeta = ReturnType<typeof getTestSuiteMeta>
 
+export type MatrixOptions = {
+  skipDBSetup?: boolean
```

Comment:
Maybe we can merge the two into a single `skipDb`? If you had it setup I think it's likely that you'll want teardown. wdyt?

Reply:
Sure, I have merged them!